tests/layer_formatting_tests.py

```diff
@@ -7,6 +9,15 @@
 
 class LayerFormattingTests(XMLBuilderMixin, TestCase):
     def test_build_header(self):
+        """
+        I think this header is supposed to look like this:
+
+
+        |1-1|1-2                    |
+        |   |2-1        |2-2        |
+        |   |3-1|3-2|3-3|3-4|3-5|3-6|
```

Comment:
Indeed -- I was thinking more about a method of checking your solution concisely than the actual output. The comment makes it clear what you want to see, much more clearer than the asserts... so maybe there's a way to skip the asserts in favor of checking the comment, if that makes sense?

It certainly doesn't seem trivial to me, but I maybe that stirs something for you? If not, no worries